### ISSUE TITLE
What's the scale of two loss when training KL loss

### ISSUE THREAD

**Issue by hding2455** [yihui-he/KL-Loss#2]
Sorry for bothering. I am confused when reading source code to find this, so I just asked directly.
when trainning total_loss = cls_loss + a*reg_loss(KL loss)
What's the number of a  did you choose to train?

**Comment by ethanhe42** [comment]
This seems a general faster R-CNN question. You can ask it in [Detectron](https://github.com/facebookresearch/Detectron). We didn't modify this balancing term.
